styles.css

```diff
@@ -0,0 +1,465 @@
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0%;
+  font-family: 'Lato', sans-serif;
+}
+
+.fadeInAnimation {
+  opacity: 1;
+  animation-name: fadein;
+  animation-duration: 2s;
+  animation-timing-function: ease-out;
+  animation-direction: alternate;
+}
+
+.delay3 {
+  animation-duration: 3s;
+}
+
+.delay5 {
+  animation-duration: 5s;
+}
+
+@keyframes fadein {
+  0% {
+    opacity: 0;
+  }
+  /* stylelint-disable-next-line rule-empty-line-before */
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes slidein {
+  from {
+    margin-left: 100%;
+    width: 300%;
+  }
+
+  to {
+    margin-left: 0%;
+    width: 100%;
+  }
+}
+
+main p {
+  font-size: 0.8rem;
+  color: #515151;
+  padding: 0.4rem 0%;
+}
+
+.small-navbar {
+  display: flex;
+  justify-content: end;
+  gap: 1.4rem;
+  padding: 0% 5rem;
+  background-color: #3e3e3e;
+}
+
+.small-navbar img {
+  max-width: 0.5rem;
+  margin: 0%;
+}
+
+.small-navbar p {
+  font-size: 0.8rem;
+  margin: 0%;
+  color: #fff;
+}
+
+nav {
+  display: flex;
+  width: 100%;
+  justify-content: space-around;
+  align-items: center;
+  padding: 0% 0.8rem;
+  box-shadow: 1px 5px 3px #3e3e3e;
+  background-color: #fff;
+  z-index: 3;
+  animation-name: slidein;
+  animation-duration: 1.5s;
+}
+
+.activeNav { position: fixed; }
+
+.nav-menu {
+  display: none;
+  list-style-type: none;
+  gap: 3rem;
+  padding: 0%;
+}
+
+.nav-menu a {
+  text-decoration: none;
+  color: #838080;
+  font-weight: 400;
+}
+
+.activeNavMenu a {
+  text-decoration: none;
+  color: #f24822;
+  font-weight: 400;
+}
+
+.activeNavMenu {
+  margin-top: 3rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  position: fixed;
+  background-color: #515151;
+  font-size: 2rem;
+  height: 90vh;
+  width: 100%;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  text-decoration: none;
+  list-style-type: none;
+}
+
+#about {
+  color: #f24822;
+}
+
+.container {
+  padding: 1rem;
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+  cursor: pointer;
+  gap: 5px;
+  left: 0%;
+}
+
+.activehamburger {
+  position: fixed;
+  z-index: 3;
+}
+
+span {
+  display: block;
+  width: 33px;
+  height: 2px;
+  background-color: #f24822;
+  border-radius: 3px;
+  transform-origin: 0% 100%;
+  transition: all 300ms;
+}
+
+.hamburger span {
+  display: block;
+  width: 33px;
+  height: 4px;
+  background: #515151;
+  border-radius: 3px;
+  transform-origin: 0% 100%;
+  transition: all 300ms;
+}
+
+.bar1 {
+  display: flex;
+}
+
+.activebar1 {
+  transform: rotate(45deg) translate(-2px, 2px);
+}
+
+.activebar2 {
+  opacity: 0;
+  margin-left: -30px;
+}
+
+.activebar3 {
+  transform: rotate(-45deg) translate(-4px, 4px);
+}
+
+/* <a href="https://www.freepik.com/free-photo/dark-concrete-texture-background_9053906.htm#query=texture%20background&position=10&from_view=keyword&track=ais">Image by kbza</a> on Freepik */
```

Comment:
- [ ] It seems like this commented code is not really useful to this file, kindly remove it for your code base to be clean. (**The same is applicable to about.css line 191**)